sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/PhoneNumberIntegrationTestBase.java

```diff
@@ -16,54 +16,46 @@
+            .get("COMMUNICATION_SERVICE_ACCESS_KEY", "QWNjZXNzS2V5");
+    private static final String ENV_ENDPOINT = Configuration.getGlobalConfiguration()
+            .get("COMMUNICATION_SERVICE_ENDPOINT", "https://REDACTED.communication.azure.com");
+    private static final String CONNECTION_STRING = Configuration.getGlobalConfiguration().get(
+            "COMMUNICATION_LIVETEST_CONNECTION_STRING",
+            "endpoint=https://REDACTED.communication.azure.com/;accesskey=QWNjZXNzS2V5");
+
+    protected static final String PHONE_NUMBER = Configuration.getGlobalConfiguration()
+            .get("COMMUNICATION_PHONE_NUMBER", "+11234567891");
+    protected static final String COUNTRY_CODE = Configuration.getGlobalConfiguration().get("COUNTRY_CODE", "US");
+    protected static final String AREA_CODE = Configuration.getGlobalConfiguration().get("AREA_CODE", "619");
+    protected static final String LOCALE = Configuration.getGlobalConfiguration().get("LOCALE", "en-us");
+    protected static final String LOCATION_OPTION_STATE = Configuration.getGlobalConfiguration()
+            .get("LOCATION_OPTION_STATE", "CA");
+    protected static final String LOCATION_OPTION_CITY = Configuration.getGlobalConfiguration()
+            .get("LOCATION_OPTION_CITY", "NOAM-US-CA-SD");
+    protected static final String RESERVATION_OPTIONS_DESCRIPTION = Configuration.getGlobalConfiguration()
+            .get("RESERVATION_OPTIONS_DESCRIPTION", "testReservation20200014");
+    protected static final String RESERVATION_OPTIONS_NAME = Configuration.getGlobalConfiguration()
+            .get("RESERVATION_OPTIONS_NAME", "testReservation20200014");
+
+    private static final StringJoiner JSON_PROPERTIES_TO_REDACT = new StringJoiner("\":\"|\"", "\"", "\":\"")
+            .add("phonePlanGroupId").add("phonePlanId");
+
+    private static final Pattern JSON_PROPERTY_VALUE_REDACTION_PATTERN = Pattern.compile(
+            String.format("(?:%s)(.*?)(?:\",|\"})", JSON_PROPERTIES_TO_REDACT.toString()), Pattern.CASE_INSENSITIVE);
 
     protected PhoneNumberClientBuilder getClientBuilder(HttpClient httpClient) {
         if (getTestMode() == TestMode.PLAYBACK) {
             httpClient = interceptorManager.getPlaybackClient();
         }
 
         PhoneNumberClientBuilder builder = new PhoneNumberClientBuilder();
-        builder
-            .httpClient(httpClient)
-            .endpoint(ENV_ENDPOINT)
-            .accessKey(ENV_ACCESS_KEY);
+        builder.httpClient(httpClient).endpoint(ENV_ENDPOINT).accessKey(ENV_ACCESS_KEY);
 
         if (getTestMode() == TestMode.RECORD) {
             List<Function<String, String>> redactors = new ArrayList<>();
             redactors.add(data -> redact(data, JSON_PROPERTY_VALUE_REDACTION_PATTERN.matcher(data), "REDACTED"));
-            builder.addPolicy(interceptorManager.getRecordPolicy(redactors));
+            builder.addPolicy(interceptorManager.getRecordPolicy());
```

Comment:
Why are we not passing the redactors anymore, same in line 76? We need to sanitize the recordings.

Reply:
The compile told me to do so. :-) Also checked other code, it is no longer required

Reply:
Let me figure out how sanitization works here.

Reply:
Pretty sure redactors still needed to be passed in. Make sure that you are using the right verison of azure-core-test, it was recently updated to take in redactor.
`<artifactId>azure-core-test</artifactId>
<version>1.6.0-beta.1</version>`

Reply:
Because it is a beta version. It is not consistent with anything else in this repository :-)

Reply:
Added a rule to the interceptorManager and hope this works. All tests are passing. Do we have test case to cover this? Nothing breaks! That's no good.

Reply:
This is redacting the payload response. I dont think we can write tests for that. Instead, we should be regenerating all the json responses and making sure that the redaction is the same as before.

Reply:
Also, is Android SDK not supposed to use beta versions?

Reply:
Also, I am not sure what you mean it is inconsistent with anything else in the repo? Can you elaborate?

Reply:
I'm trying to run the tests in "RECORD" mode to see if anything changed but currently having technical issues and Jorge is helping me out.

"it is inconsistent with anything else in the repo": I searched pom.xml files for references to "azure-core-test" and this pom.xml in the PR is the only one referencing 1.6.0-beta.1. Everyone else is referencing 1.5.1.